a3fe/run/leg.py

```diff
@@ -673,9 +672,10 @@ def setup_stages(
                 property_map={"velocity": "foo"},
             )  # We will run outside of BSS
 
-            # Copy input written by BSS to the stage input directory
+            # Copy input written by BSS to the stage input directory, excluding only somd.cfg
             for file in _glob.glob(f"{stage_input_dir}/lambda_0.0000/*"):
```

Comment:
I think this would be clearer (please modify as required)

files = [
    file for file in glob.glob(f"{stage_input_dir}/lambda_0.0000/*")
    if not file.endswith(".cfg")
]